pkgs/applications/networking/libcoap/default.nix

```
@@ -0,0 +1,31 @@
+{ fetchFromGitHub, automake, autoconf, which, libtool, stdenv }:
+stdenv.mkDerivation {
+  pname = "libcoap";
+  version = "4.2.0";
+  src = fetchFromGitHub {
+    repo = "libcoap";
+    owner = "obgm";
+    rev = "v4.2.0";
```

This will make updates easier:

```suggestion
    rev = "v{version}";
```